### ISSUE TITLE
Linking with bgfx

### ISSUE THREAD

**Issue by Constan7ine** [puffi/bgfx-d#3]
Hello, I'm trying to get these bindings to work. 

I assume you need to statically link the bgfx C++ library with optilink when making your D exe. 

However when I do so I get the message " Error 43: Not a Valid Library File ".

What is the correct usage of these bindings?

**Comment by puffi** [comment]
Yes, on Windows you need to statically link, as no symbols are exported. Or use the C-API.
Did you compile bgfx with Visual Studio? If so, you should probably use that linker for your D project too. But can't help you much, sorry, I don't use Windows.
